server/@types/express/index.d.ts

```diff
@@ -60,7 +59,6 @@ declare module 'express-session' {
     affectAbilityToWorkForm: AffectAbilityToWorkForm
     reasonsNotToGetWorkForm: ReasonsNotToGetWorkForm
     workInterestTypesForm: WorkInterestTypesForm
-    workInterestRolesForm: WorkInterestRolesForm
```

Comment:
Because the Work Interests Roles controllers (both Update and Create) do not validate the form submission (there is nothing to validate because all the fields are optional), there is no need to potentially hold the form on the session. So I've taken the opportunity to tidy this all up 👍